### PR TITLE
Grammar, spelling and style fixes

### PR DESCRIPTION
**Pull request by straight-shoota**

### CODE REVIEW

File: docs/getting_started/cli.md [hunk 1]
```diff
@@ -372,16 +372,16 @@ You may find a list of **available colors** and **text decorations** in the [API
...
+Right now the code containing the logic of each of the applications always gets executed with the `OptionParser`, i.e. there is no way to include that file without running the whole application. So first we would need to refactor the code, separating the code necessary for parsing options from the logic. Once the refactoring is done, we could start testing the logic and including the file with the logic in the testing files we need. We leave this as an exercise for the reader.
 
 ## Using `Readline` and `NCurses`
 
-In case we want to build richer CLI applications, there are libraries that can help us. Here we will name two well-known libraries: `Readline` and `NCurses`.
+In case we want to build richer CLI applications, some libraries can help us. Here we will name two well-known libraries: `Readline` and `NCurses`.
```
Comment (oprypin) [inline docs/getting_started/cli.md:381]: Previous text was correct and better

File: docs/tutorials/basics/40_strings.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Strings
 
-In the previous lessons we have already made an acquaintance with a major building block
+In the previous lessons, we have already made an acquaintance with a major building block
 of most programs: strings. Let's recapitulate the basic properties:
 
 A [string](https://en.wikipedia.org/wiki/String_(computer_science)) is a sequence of [Unicode](https://en.wikipedia.org/wiki/Unicode) characters encoded in [UTF-8](https://en.wikipedia.org/wiki/UTF-8).
 A string is [immutable](https://en.wikipedia.org/wiki/Immutable_object):
-If you apply a modification to a string, you actually get a new string with the
+If you apply a modification to a string, you get a new string with the
```
Comment (oprypin) [inline docs/tutorials/basics/40_strings.md:8]: Previous text was correct and better

File: docs/tutorials/basics/40_strings.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ puts "I say: \"Hello \\\n\tWorld!\""
 
 ### Alternative Delimiters
 
-Some string literals may contain a lot of double quotes – think of HTML tags with quoted argument values for example. It would be cumbersome to have to escape each one with a backslash. Alternative literal delimiters are a convenient alternative. `%(...)` is equivalent to `"..."` except that the delimiters are denoted by parentheses (`(` and `)`) instead of double quotes.
+Some string literals may contain a lot of double quotes – think of HTML tags with quoted argument values for example. It would be cumbersome to have to escape each one with a backslash. Alternative literal delimiters are a convenient alternative. `%(...)` is equivalent to `"..."` except that the delimiters are denoted by parentheses (`(` and `)`) instead of double-quotes.
```
Comment (oprypin) [inline docs/tutorials/basics/40_strings.md:55]: New text is wrong, old text is correct.

File: docs/tutorials/basics/50_control_flow.md [hunk 4]
```diff
@@ -73,7 +73,7 @@ p! "foo" && nil,
    "bar" || "foo"
 ```
 
-The `NOT`, `XOR` and equivalence operators always return a `Bool` value (`true` or `false`).
+The `NOT`, `XOR`, and equivalence operators always return a `Bool` value (`true` or `false`).
```
Comment (oprypin) [inline docs/tutorials/basics/50_control_flow.md:76]: I am against this `, and` business. I think most people prefer to never use a comma before "and".

Comment (straight-shoota) [reply docs/tutorials/basics/50_control_flow.md:76]: In this case, it's not strictly necessary, true. But in others, it helps avoid ambiguity.

Comment (oprypin) [reply docs/tutorials/basics/50_control_flow.md:76]: Not that it's relevant in any way, but the reason it's particularly jarring to me is that in Ukrainian language `and` means `and`, while `, and` means `and then`.

Don't see how it prevents ambiguity.

Either way, not gonna push this one.

**Review by oprypin — APPROVED** [review]

File: docs/tutorials/basics/50_control_flow.md (hunk 4 above)
Comment (straight-shoota) [reply docs/tutorials/basics/50_control_flow.md:76]: https://en.wikipedia.org/wiki/Serial_comma#Ambiguity

**Pull request merged**